### PR TITLE
docs: absolute links are better for local use

### PR DESCRIPTION
**Pull request by joncodo**
Fixes: #786 https://github.com/merico-dev/lake/issues/786

### CODE REVIEW

File: README-zh-CN.md [hunk 1]
```diff
@@ -118,9 +118,9 @@ Dev Lake 是一个研发效能分析平台，它通过对软件开发生命周
 
 目录 | 内容 | 文档
 ------------ | ------------- | -------------
-Jira | 概述，数据和指标，配置，API | [Link](plugins/jira/README-zh-CN.md) 
-Gitlab | 概述，数据和指标，配置，API | [Link](plugins/gitlab/README-zh-CN.md) 
-Jenkins | 概述，数据和指标，配置，API | [Link](plugins/jenkins/README-zh-CN.md) 
+Jira | 概述，数据和指标，配置，API | [Link](plugins/jira/README-zh-CN.md)
```
[the remark this answers is not in the document]
Comment (joncodo) [reply README-zh-CN.md:121]: Yes. Thanks. I will update

**Comment by kevin-kline** [comment]
@joncodo I think these need changing too:

image

**Comment by kevin-kline** [comment]
and this one:

image

**Review by kevin-kline — COMMENTED** [review]
Some changes still missing. See comments in main thread.

**Comment by klesh** [comment]
Wait a sec, looks like we are setting up mirror repo on gitee.com, absolute link is not ideal in that case.
Just pause until I figure it out.

**Pull request closed without merging**